ui/page/security/validate_address.go

```diff
@@ -32,8 +33,7 @@ type ValidateAddressPage struct {
 	addressEditor         decredmaterial.Editor
 	clearBtn, validateBtn decredmaterial.Button
 	stateValidate         int
-	walletName            string
-	isEnabled             bool
+	_                     string
```

Comment:
This variable was declared but not used. Also, I am not sure why it was created.